lib/puppet/resource/status.rb

```diff
@@ -83,6 +83,15 @@ class Status
       #     while evaluating `@real_resource`.
       attr_reader :events
 
+      # @!attribute [rw] dependency_failed
+      #   @return [Boolean] A cache of whether some dependency of this
```

Comment:
I copying the style of the attributes above. I'm not sure offhand whether that indentation is semantically meaningful or not to YARD.